### ISSUE TITLE
"config -set sensitivity=" does not work

### ISSUE THREAD

**Issue by Allofich** [joncampbell123/dosbox-x#2910]
For example, `config -set sensitivity=80` does not work. The mouse sensitivity remains unchanged. It sensitivity can be changed through the menu. `config -set ver=` does work, so not all `config -set` commands are failing.

Tested with 88370683291f5c681d2a7d6fff088b947b783bd4 on both the SDL1 and SDL2 64-bit Visual Studio 2019 builds, on Windows 10.

**Comment by Allofich** [comment]
`config -get sensitivity` shows the value as having been updated to whatever was used with `set`, but if you select "DOS > Mouse emulation > Sensitivity" the value used with `set` does not show. The actual mouse sensitivity is also unchanged (even if you use `config -set sensitivity` with a very large or small value, the actual in-program mouse sensitivity is unchanged)

**Comment by grapeli** [comment]
The ability to dynamically change the sensitivity of the mouse via the dosbox-x menu is much more useful and more important. We want to quickly change this value when we use a running game or program.
Changing via config is less important. We might as well restart dosbox-x into the shell again quickly.
`dosbox-x -set sensitivity=50`

**Comment by Allofich** [comment]
True, I have never even used this feature before. It is a bug, just a minor one.
